test/unit/Wizard.unit.js

```diff
@@ -563,9 +563,13 @@ describe('Wizard Form with Nested Form validation', () => {
           setTimeout(() => {
             checkPage(2);
             const errors = wizard.errors;
-            assert.equal(errors.length, 1, 'Must err before next page');
+            assert.equal(errors.length, 2, 'Must err before next page');
```

Comment:
the nested wizard has two required fields on the same page:

1. formNested.data.textField
2. formNested.data.dataGrid[0].textField

both are required and should show up in the errors, I believe

Reply:
this commit in core added a new check in eachComponentData that causes both errors to show

https://github.com/formio/core/commit/fc8960ecb5460a613c513916c8bab7d2b02c1721#diff-9b18bb8f4f70621ee3d4d779a4293139a5ede3ac9d714d46560027a5e0750faaR86